services/create_project.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import argparse
+import sys
+import click
+from typing import AnyStr
+
+sys.path.append("../")
+from models import *
+
+
+def create_client(name: AnyStr) -> None:
+    proj_response = requests.post(
+        f"{V1_API_URL}/workspaces/{WORKSPACE_ID}/clients",
+        json={"name": name},
+        headers=HEADERS,
+    )
+    print(f"Client {name} created.")
+    pass
+
+
+@click.command()
+@click.option("--name", prompt="Project name", help="The name for the project.")
+# TODO: implement clients in our DB
+def create_project(name: AnyStr) -> None:
+    colors = {
+        "e": "#69b39e",
+        "t": "#09215d",
+        "n": "#FF9800",
```

Comment:
Not sure if this changes something, but this is how it is in clockify:

```suggestion
        "n": "#ff9800",
```